package.json

```diff
@@ -18,7 +18,11 @@
     "effective:config": "circleci config process .circleci/config.yml | sed /^#/d"
   },
   "peerDependencies": {
-    "cypress": "*"
+    "@babel/core": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "babel-loader": "^8.0.0",
+    "cypress": "*",
+    "webpack": "^5.0.0"
```

Comment:
Should this be `^4.0.0` @CharlesStover? That should also be fine.

Also you should either have`babel-loader` OR `@babel-loader`, do we need to list both of these?

Reply:
Actually it looks like this is OUR fault, we ask for those versions of those modules. That seems wrong, but it's not really something to fix in this PR.

I think you need to set webpack to `^4 || ^5`, though.